docs/assets/1.25-2.0-remove-deprecated-resources.sh

```diff
@@ -0,0 +1,229 @@
+kubectl -n kyma-system delete role service-catalog-addons-service-catalog-ui
+kubectl -n kyma-system delete role apiserver-proxy-certs-job
+kubectl -n kyma-system delete role apiserver-proxy-certs-job-gardener-certs-role
+kubectl -n kyma-system delete role apiserver-proxy-ssl-helper-role
+kubectl -n kyma-system delete role compass-runtime-agent-tests-dex-secrets
+kubectl -n kyma-system delete role ory-mechanism-migration
+kubectl -n kyma-system delete role ory-oathkeeper-keys-helper-job-role
+kubectl -n kyma-integration delete role application-connector-tests
+kubectl -n kyma-integration delete role application-operator-tests
+kubectl -n istio-system delete role application-connector-certs-sync
+kubectl -n istio-system delete role istio-kyma-validate
+
+echo "
+Deleting orphaned RoleBindings"
+kubectl -n kyma-system delete rolebinding logging-log-ui
+kubectl -n kyma-system delete rolebinding helm-broker-addons-ui
+kubectl -n kyma-system delete rolebinding service-catalog-addons-service-catalog-ui
+kubectl -n kyma-system delete rolebinding apiserver-proxy-certs-job
+kubectl -n kyma-system delete rolebinding apiserver-proxy-certs-job-gardener-certs-role
+kubectl -n kyma-system delete rolebinding apiserver-proxy-ssl-helper-role-binding
+kubectl -n kyma-system delete rolebinding compass-runtime-agent-tests-dex-secrets
+kubectl -n kyma-system delete rolebinding ory-mechanism-migration
+kubectl -n kyma-system delete rolebinding ory-oathkeeper-keys-helper-job-role-binding
+kubectl -n kyma-system delete rolebinding apiserver-proxy-certs-job-kube-system-role
+kubectl -n istio-system delete rolebinding application-connector-certs-sync
+kubectl -n istio-system delete rolebinding istio-kyma-validate
+kubectl -n kyma-integration delete rolebinding application-connector-tests
+kubectl -n kyma-integration delete rolebinding application-operator-tests
+
+echo "
+Deleting orphaned Secrets"
+kubectl -n kyma-system delete secret test-developer-user
+kubectl -n kyma-system delete secret test-namespace-admin-user
+kubectl -n kyma-system delete secret test-no-rights-user
+kubectl -n kyma-system delete secret test-read-only-user
+kubectl -n kyma-system delete secret admin-user
+kubectl -n kyma-system delete secret apiserver-proxy-tls-cert
+
+echo "
+Deleting orphaned ConfigMaps"
+kubectl -n kyma-system delete configmap addons-ui
+kubectl -n kyma-system delete configmap logging-log-ui
+kubectl -n kyma-system delete configmap service-catalog-ui
+kubectl -n kyma-system delete configmap apiserver-proxy
+kubectl -n kyma-system delete configmap cluster-essentials-crd-0
+kubectl -n kyma-system delete configmap cluster-essentials-crd-1
+kubectl -n kyma-system delete configmap cluster-essentials-crd-2
+kubectl -n kyma-system delete configmap cluster-users
+kubectl -n istio-system delete configmap kyma-extra-manifests
+kubectl -n istio-system delete configmap kyma-istio-operator-config
+kubectl -n istio-system delete configmap kyma-istio-operator-config-backup
```

Comment:
```suggestion
kubectl -n istio-system delete configmap kyma-istio-operator-config-backup
kubectl -n istio-system delete configmap rafter-minio
```

Reply:
@a-thaler  please verify this once

Reply:
I still see it in the helm chart, so I assume it is still needed